libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ConfusionMatrixHeatmap.tsx

```diff
@@ -176,7 +176,7 @@ export class ConfusionMatrixHeatmap extends React.Component<
                     marginTop: 80,
                     plotBorderWidth: 1,
                     type: "heatmap",
-                    width: this.state.selectedClasses.length * 40 + 200
+                    width: this.state.selectedClasses.length * 100 + 200
```

Comment:
Could you please confirm if it works fine with 320*256px size? Thanks.

Reply:
If I remember correctly, 2D information is not required for reflow. This works the same as other tables, when it is too small it will have a horizontal scrollbar. Please let me know if it does not make sense. Thanks!